models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml

```diff
@@ -0,0 +1,23 @@
+models:
+  - name: machine-translation-nar-en-de-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: preds
+        inputs:
+          - name: "tokens"
+            value: 'tokens'
+            type: INPUT
+
+    datasets:
+      - name: WMT_en_de
+        postprocessing:
```

Comment:
Please add preprocessing

Reply:
I wrote this file like in machine-translation-nar-en-ru: https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/machine-translation-nar-en-ru-0001/accuracy-check.yml
There aren't any preprocessing steps there. Where can I find the documentation for preprocessing description? Thanks!

Reply:
There are preprocessing steps in dataset definitions. Resolved

Reply:
@Anna-Mironova is it looks good for you now?